### ISSUE TITLE
NuGet Problem

### ISSUE THREAD

**Issue by SteveHindmarsh** [coni2k/DbContextGeneratorWithCodeFirst#2]
I've tried to make this solution build by ensuring all the files are unblocked and not read only and trying all the suggestions here: https://docs.microsoft.com/en-gb/nuget/Consume-Packages/Package-restore-troubleshooting

But still get this exception from VS2017

Severity	Code	Description	Project	File	Line	Suppression State
Error		This project references NuGet package(s) that are missing on this computer. Enable NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is D:\SourceCodeDemos\DbContextGeneratorWithCodeFirst-master\\.nuget\NuGet.targets.	DbContextGeneratorWithCodeFirst	D:\SourceCodeDemos\DbContextGeneratorWithCodeFirst-master\DbContextGeneratorWithCodeFirst\DbContextGeneratorWithCodeFirst.csproj	88	

I cured this problem by removing the following from the project file:
DbContextGeneratorWithCodeFirst.csproj

```
  <Import Project="$(SolutionDir)\.nuget\NuGet.targets" Condition="Exists('$(SolutionDir)\.nuget\NuGet.targets')" />
  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
    <PropertyGroup>
      <ErrorText>This project references NuGet package(s) that are missing on this computer. Enable NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
    </PropertyGroup>
    <Error Condition="!Exists('$(SolutionDir)\.nuget\NuGet.targets')" Text="$([System.String]::Format('$(ErrorText)', '$(SolutionDir)\.nuget\NuGet.targets'))" />
  </Target>
```

**Comment by coni2k** [comment]
I need to look at the issue in detail, but I'm only going to be available next week.
________________________________

**Comment by coni2k** [comment]
@SteveHindmarsh hi,
Thank you for reporting this.

Just had a time to look at it, you are right, I had to remove that block in order to make it work with Nuget again.

Updated the project file accordingly, also fixed another issue with new Visual Studio paths in "EF6.Utility.CS.ttinclude" file.

Master contains the latest version and should be working out of the box.
